internal/envoy/install_test.go

```diff
@@ -162,6 +162,20 @@ func TestInstallIfNeeded(t *testing.T) {
 	require.Contains(t, out.String(), "100% |████████████████████████████████████████|")
 }
 
+func TestInstallIfNeeded_NotFound(t *testing.T) {
```

Comment:
@mathetake here are the tests that show how good life is when the graph isn't nested pointers anymore!

Reply:
super!